pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -247,12 +262,7 @@ let
     installCheckPhase = attrs.checkPhase;
   } //  lib.optionalAttrs (disabledTestPaths != []) {
       disabledTestPaths = lib.escapeShellArgs disabledTestPaths;
-  }));
-
-  passthru.updateScript = let
-      filename = builtins.head (lib.splitString ":" self.meta.position);
-    in attrs.passthru.updateScript or [ update-python-libraries filename ];
-in lib.extendDerivation
-  (disabled -> throw "${name} not supported for interpreter ${python.executable}")
-  passthru
-  self
+  };
+in
+assert (disabled -> throw "${name} not supported for interpreter ${python.executable}");
```

Comment:
This evaluates the check more eagerly than it used to. `extendDerivation` only added it to the outputs and `drvPath`.
This will run the checks before returning the result, which makes the check less lazy, which could result in performance loss and more infinite recursions.

Reply:
It turns out that I forgot to consider the `modifier` after `overrideAttrs`. Maybe it was the cause for the original method not to work.

Just corrected it and applied the original disabling approach.